### PR TITLE
3 feature 5 be able to win a round

### PR DESCRIPTION
**Pull request by ericshi03**
Added be able to win a round function as well as some helpers

### CODE REVIEW

**Review by enfz — APPROVED** [review]
The logic looks good for the algorithm. Will need more Javadoc throughout the program and change packages for the files. Like the other Player and Game classes, this will need to be implemented cleanly with the use of factories.

**Review by Jefferson-liu — APPROVED** [review]
The logic for the game and player classes looks fine. We should probably remove the main method from the game class and put it into its designated class. Otherwise, everything looks fine.

File: src/main/java/tutorial/Game.java [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+        } else if (isStraight) {
+            return 52 + (straightMax != 1 ? straightMax - 1 : 13);
+        } else if (is3Same) {
+            return 39 + (trios.get(0) != 1 ? trios.get(trios.size() - 1) - 1 : 13);
+        } else if (pairs.size() >= 2) {
+            return 26 + (pairs.get(0) != 1 ? pairs.get(pairs.size() - 1) - 1 : 13); //this is also broken
+        } else if (is2Same) {
+            return 13 + (pairs.get(0) != 1 ? pairs.get(pairs.size() - 1) - 1 : 13);
+        }
+
```
Comment (xiaoxinamek) [inline src/main/java/tutorial/Game.java:185]: Code looks good,  should be no problem after this part been fixed. It is great that you have add test while you are writing it.

**Review by xiaoxinamek — APPROVED** [review]
Code looks good, should be no problem after this part been fixed. It is great that you have add test while you are writing it.

**Review by Jefferson-liu — COMMENTED** [review]
Also package needs to be changed from tutorial to entities or something

**Pull request merged**